src/isar/config/logging.conf

```diff
@@ -21,16 +16,24 @@ handlers:
 loggers:
   console:
     level: DEBUG
-    handlers: [console]
+    handlers: []
     propagate: no
   api:
     level: DEBUG
-    handlers: [api, console]
+    handlers: [api]
     propagate: no
-  state_machine:
+  urllib3:
+    level: WARNING
+    handlers: []
+  uvicorn:
     level: DEBUG
-    handlers: [state_machine, console]
+    handlers: []
     propagate: no
+  state_machine:
+    level: DEBUG
+    handlers: [state_machine]
+    propagate: False
 root:
   level: DEBUG
-  handlers: [console]
\ No newline at end of file
+  handlers: []
+  
```

Comment:
Ther should be a newline but nothing else. My guess is that there is a space or something on line 39 which is screwing this up?